bpf/bpf_lxc.c

```diff
@@ -866,7 +888,13 @@ __section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_IPV6_TO_LXC) int tail_ipv6_policy(s
 	__u32 src_label = skb->cb[CB_SRC_LABEL];
 	int forwarding_reason = 0;
 
-	ret = ipv6_policy(skb, ifindex, src_label, &forwarding_reason);
+	struct ep_config *cfg = lookup_ep_config();
+
+	if(cfg)
+	    ret = ipv6_policy(skb, ifindex, src_label, &forwarding_reason, cfg);
```

Comment:
nit: spacing

(You can likely search for instances of 2 or more spaces and they likely should be substituted for tabs)